lib/gollum-lib/macro/video.rb

```diff
@@ -0,0 +1,17 @@
+module Gollum
+  class Macro
+    class Video < Gollum::Macro
+      def render (fname)
+        begin 
```

Comment:
Well it is now simplified to a bare minimum for what is needed to link to mp4 files in the wiki. We should recommend mp4 as this is the widely used format and all browsers support it.